CASC/Schedules.hpp

```diff
@@ -27,22 +27,25 @@ class Schedules
 public:
   static void getScheduleFromFile(const std::string& filename, Schedule& quick);
 
-  static void getCasc2024Schedule(const Shell::Property& property, Schedule& quick, Schedule& fallback);
-  static void getCascSat2024Schedule(const Shell::Property& property, Schedule& quick, Schedule& fallback);
+  static void getCasc2024Schedule(const Shell::Property& property, Schedule& quick);
+  static void getCascSat2024Schedule(const Shell::Property& property, Schedule& quick);
 
-  static void getSmtcomp2018Schedule(const Shell::Property& property, Schedule& quick, Schedule& fallback);
+  static void getCasc2025Schedule(const Shell::Property& property, Schedule& quick, Schedule& champions);
+  static void getCascSat2025Schedule(const Shell::Property& property, Schedule& quick, Schedule& champions);
+
+  static void getSmtcomp2018Schedule(const Shell::Property& property, Schedule& quick);
 
   static void getLtb2017Hh4Schedule(const Shell::Property& property, Schedule& sched);
   static void getLtb2017IsaSchedule(const Shell::Property& property, Schedule& sched);
   static void getLtb2017HllSchedule(const Shell::Property& property, Schedule& sched);
   static void getLtb2017MzrSchedule(const Shell::Property& property, Schedule& sched);
   static void getLtb2017DefaultSchedule(const Shell::Property& property, Schedule& sched);
 
-  static void getInductionSchedule(const Shell::Property& property, Schedule& quick, Schedule& fallback);
```

Comment:
Is there a pithy reason why we don't have fallback schedules any more? Or why we had them in the first place? Just curious.

Reply:
So the short answer is. I killed `fallback`, but newly use the second `Schedule&` for this new thing, semantically.